workflows/prognostic_c48_run/tests/_regtest_outputs/test_regression.test_fv3run_diagnostic_outputs_check_variables[nudging].out

```diff
@@ -1,30 +1,31 @@
 air_temperature_reference: 83b835d90ea5b4b70938f6a3b744a052
-air_temperature_tendency_due_to_nudging: 74cc6401aef31920fc1abed6b000c5cc
+air_temperature_tendency_due_to_nudging: d461b6836c0c43116df5d661f126bfd9
 area: f31fc363daec5c8e5f76f2762412deb5
-cnvprcp_after_physics: 18dd5bdf6fb264bf9342558240cb5bb9
-cnvprcp_after_python: 18dd5bdf6fb264bf9342558240cb5bb9
-column_heating_due_to_nudging: 6c95d9c76803e619636fdf4700965a22
-evaporation: aeaa17fdb186a9bfb1cb4476f2946d78
-net_moistening_due_to_nudging: 2138071ad62572e69ae5448316d7aa32
-physics_precip: 6130e940fdc375fd5740be268e1f3cf2
+cnvprcp_after_physics: 07932d60c92186bd2093affebec0ab8d
```

Comment:
Lots of checksum changes here, which makes it tricky to review and ensure that existing configurations aren't being broken by your new feature.  If you find something unappealing about the existing regression configurations could you isolate the changes causing this to their own PR? Thanks.

Reply:
The fact that the `predictor` config checksums do not change is good evidence that other features (diagnostics, etc) are not breaking IMO.

Reply:
If you feel strongly about this I can make a separate PR. I just added tendency prescribing to the nudged run to test it in integration with our other pieces. Similar to the `emulator` getting added to the regression tests previously.

Reply:
ok. I think the risk is low in this case given that the other tests pass, but I didn't bring it up for no reason. This is not a forgiving part of the code...a single import out of order can cause the model to crash and consume hours of debugging time. This happens regularly, and I've relied heavily on `git log/bisect` to track these problems down. Keeping the git history clean is worth the 5 minutes it would take to split the checksum breaking changes into a new PR or interactively rebase your branch to be a bit cleaner do so that you can "rebase and merge" rather than "squash and merge".

Reply:
Thanks, yeah I agree care is required here. I just think for this PR given the small area of overlap with `loop.py` and the nudging code, we are pretty safe.